css-brochure-form/styles.css

```diff
@@ -63,10 +58,9 @@ input {
 }
 
 .button {
-  text-align: 80%;
   text-transform: uppercase;
   margin-top: 8px;
-  position: absolute;
-  right: 555px;
+  position: relative;
+  left: 245px;
```

Comment:
This is just the same problem from a different side. You are pushing the element where you want it to be, which will break colossally when mobile responsiveness comes into play. You should use `text-align: right` to move the button to the right-hand side of the `div`, that will get you the same behavior in a way that works across all screen sizes.